.travis.yml

```diff
@@ -8,4 +8,4 @@ rvm:
   - 2.0.0
 script: bundle exec rake test
 env:
-  - PUPPET_VERSION="3.8.0" STRICT_VARIABLES=yes FUTURE_PARSER=yes STRINGIFY_FACTS=false
```

Comment:
It doesn't affect spec tests, because we stub all the facts. I'm also not sure the `STRINGIFY_FACTS` variable actually did anything.